pt_BR.ini

```diff
@@ -3,12 +3,12 @@ Load = Carregar jogo
 Game Settings = Configurações
 Credits = Créditos
 www.ppsspp.org = www.ppsspp.org
-Support PPSSPP = Suporte PPSSPP
+Support PPSSPP = Ajude o projeto PPSSPP
```

Comment:
gonna change, thanks for the information






Sent from Windows Mail





From: papel
Sent: ‎Thursday‎, ‎February‎ ‎5‎, ‎2015 ‎12‎:‎37‎ ‎AM
To: hrydgard/ppsspp-lang
Cc: Gabriel Franco





In pt_BR.ini:
> @@ -3,12 +3,12 @@ Load = Carregar jogo
>  Game Settings = Configurações
>  Credits = Créditos
>  www.ppsspp.org = www.ppsspp.org
> -Support PPSSPP = Suporte PPSSPP
> +Support PPSSPP = Ajude o projeto PPSSPP


The new text is longer and overlaps the gold icon.

—
Reply to this email directly or view it on GitHub.